newIDE/app/src/AssetStore/index.js

```diff
@@ -308,17 +305,13 @@ export const AssetStore = ({
                           <Column expand alignItems="flex-end" noMargin>
                             <RaisedButton
                               primary
-                              label={
-                                isAssetPackAdded ? (
-                                  <Trans>Asset pack added</Trans>
-                                ) : (
-                                  <Trans>Add pack to my scene</Trans>
-                                )
-                              }
+                              label={<Trans>Add all assets to my scene</Trans>}
                               onClick={() =>
```

Comment:
The text doesn't change because we don't know wether all assets are already added or not?

Reply:
Yes, we don't know at this stage, we do the logic in the dialog that opens up